### PR TITLE
feat: display memory eviction policy in redis admin widget

### PR DESCRIPTION
**Pull request by imorland**
For example:

image

or

image

etc

### CODE REVIEW

**Comment by luceos** [comment]
Although I can understand where this comes from. I am worried that the widgets in the admin are steering towards an expert mode instead of providing noob-friendly information.

Maybe this information would make more sense in the dashboard when they would be translated towards a human readable form?

**Comment by imorland** [comment]
One could argue that `Horizon` is already an expert setup, rather than for noobs, but nonetheless, I have added a tooltip to the widget, which gives a brief explanation, as well as a [link to the official Redis documentation for data eviction policy](https://redis.io/docs/latest/develop/reference/eviction/)

image

File: js/src/admin/components/HorizonStatsWidget.tsx [hunk 1]
```diff
@@ -121,13 +128,53 @@ export default class HorizonStatsWidget extends DashboardWidget {
...
+          <span>
+            {' '}
+            <LinkButton href={infoUrl} external={true} target="_blank" icon="fas fa-info-circle" />
+          </span>
+        </Tooltip>,
+        90
+      );
+    }
+
+    return items;
```
Comment (luceos) [inline js/src/admin/components/HorizonStatsWidget.tsx:177]: This seems to be code that is not used in the PR, was this added by mistake?

**Review by luceos — CHANGES_REQUESTED** [review]

File: js/src/admin/components/HorizonStatsWidget.tsx (hunk 1 above)
Comment (imorland) [reply js/src/admin/components/HorizonStatsWidget.tsx:177]: Please look again, this code is used to construct the widget title labels (ie if a `infoLabel` and `infoUrl` are passed, the tooltip and `i` icon are then added).

It also lays the foundation of making it possible to extend and add additional widgets to the queue status from other extensions (although this is not exported yet, but it is a start)

Comment (luceos) [reply js/src/admin/components/HorizonStatsWidget.tsx:177]: Ah yea I see 👍

**Pull request merged**